ocaml/testsuite/tests/typing-layouts-bits32/stdlib_bits32_u.ml

```diff
@@ -0,0 +1,316 @@
+(* TEST
```

Comment:
This file introduces yet another implementation of QuickCheck in our testsuite. In some ways, it's better than the one for float64. But it's sad that there is no shared infrastructure here. I don't wish to delay this work behind a cleanup, but can I ask you to make a ticket in our internal system (for v2.5) to clean this up?